package.json

```
@@ -36,6 +36,8 @@
     "test:unit:watch": "npm run test:unit -- --watch",
     "test:integration": "npm run stop && npm run build:local && (npm run start & wait-on -t 20000 http://localhost:7080/status) && (jest --runInBand --colors ./src/integration/pages/**/*) ; npm run stop",
     "test:integration:ci": "jest --ci --runInBand --colors ./src/integration/pages/**/*",
+    "test:linkey": "node scripts/linkeySetup.js && npx jest src/app/lib/config/services/*.test.js --verbose true ; npm run test:linkey:cleanup",
```

I believe we don't need the `npx`
```suggestion
    "test:linkey": "node scripts/linkeySetup.js && jest src/app/lib/config/services/*.test.js --verbose true; npm run test:linkey:cleanup",
```